docker/openemr/7.0.5/upgrade/fsupgrade-1.sh

```diff
@@ -6,87 +6,91 @@ priorOpenemrVersion="5.0.1"
     sitename=$(basename "${dir}")
 
     # Ensure have all directories
     echo "Start: Ensure have all directories in ${sitename}"
-    if [ ! -d ${dir}/documents/certificates ]; then
-        mkdir -p ${dir}/documents/certificates
+    if [ ! -d "${dir}/documents/certificates" ]; then
+        mkdir -p "${dir}/documents/certificates"
     fi
-    if [ ! -d ${dir}/documents/couchdb ]; then
-        mkdir -p ${dir}/documents/couchdb
+    if [ ! -d "${dir}/documents/couchdb" ]; then
+        mkdir -p "${dir}/documents/couchdb"
     fi
-    if [ ! -d ${dir}/documents/custom_menus/patient_menus ]; then
-        mkdir -p ${dir}/documents/custom_menus/patient_menus
+    if [ ! -d "${dir}/documents/custom_menus/patient_menus" ]; then
+        mkdir -p "${dir}/documents/custom_menus/patient_menus"
     fi
-    if [ ! -d ${dir}/documents/edi ]; then
-        mkdir -p ${dir}/documents/edi
+    if [ ! -d "${dir}/documents/edi" ]; then
+        mkdir -p "${dir}/documents/edi"
     fi
-    if [ ! -d ${dir}/documents/era ]; then
-        mkdir -p ${dir}/documents/era
+    if [ ! -d "${dir}/documents/era" ]; then
+        mkdir -p "${dir}/documents/era"
     fi
-    if [ ! -d ${dir}/documents/letter_templates ]; then
-        mkdir -p ${dir}/documents/letter_templates
+    if [ ! -d "${dir}/documents/letter_templates" ]; then
+        mkdir -p "${dir}/documents/letter_templates"
     fi
-    if [ ! -d ${dir}/documents/logs_and_misc/methods ]; then
-        mkdir -p ${dir}/documents/logs_and_misc/methods
+    if [ ! -d "${dir}/documents/logs_and_misc/methods" ]; then
+        mkdir -p "${dir}/documents/logs_and_misc/methods"
     fi
-    if [ ! -d ${dir}/documents/mpdf/pdf_tmp ]; then
-        mkdir -p ${dir}/documents/mpdf/pdf_tmp
+    if [ ! -d "${dir}/documents/mpdf/pdf_tmp" ]; then
+        mkdir -p "${dir}/documents/mpdf/pdf_tmp"
     fi
-    if [ ! -d ${dir}/documents/onsite_portal_documents/templates ]; then
-        mkdir -p ${dir}/documents/onsite_portal_documents/templates
+    if [ ! -d "${dir}/documents/onsite_portal_documents/templates" ]; then
+        mkdir -p "${dir}/documents/onsite_portal_documents/templates"
     fi
-    if [ ! -d ${dir}/documents/procedure_results ]; then
-        mkdir -p ${dir}/documents/procedure_results
+    if [ ! -d "${dir}/documents/procedure_results" ]; then
+        mkdir -p "${dir}/documents/procedure_results"
     fi
-    if [ ! -d ${dir}/documents/smarty/gacl ]; then
-        mkdir -p ${dir}/documents/smarty/gacl
+    if [ ! -d "${dir}/documents/smarty/gacl" ]; then
+        mkdir -p "${dir}/documents/smarty/gacl"
     fi
-    if [ ! -d ${dir}/documents/smarty/main ]; then
-        mkdir -p ${dir}/documents/smarty/main
+    if [ ! -d "${dir}/documents/smarty/main" ]; then
+        mkdir -p "${dir}/documents/smarty/main"
     fi
-    if [ ! -d ${dir}/documents/temp ]; then
-        mkdir -p ${dir}/documents/temp
+    if [ ! -d "${dir}/documents/temp" ]; then
+        mkdir -p "${dir}/documents/temp"
     fi
```

Comment:
It might be necessary in php, but in bash there isn't any good reason to do all of these checks and then mkdir -p one at a time.

```suggestion
    mkdir -p "${dir}/documents/"{certificates,couchdb,custom_menus/patient_menus,edi,era,letter_templates,logs_and_misc/methods,mpdf/pdf_tmp,onsite_portal_documents/templates,procedure_results,smarty/gacl,smarty/main,temp}
```